File: sqlite3/perl/delete/sqlite3_delete.pl
#!/usr/bin/perl
#
#	perl/delete/sqlite3_delete.pl
#
#				Jun/08/2015
#
# ---------------------------------------------------------
use	strict;
use	warnings;
use	utf8;
use	Encode;
#
use DBI;
#
use lib '/var/www/data_base/common/perl_common';
use text_manipulate;
use sql_manipulate;
use sqlite3_manipulate;
#
# ---------------------------------------------------------
print	(encode ('utf-8',"*** 開始 ***\n"));
my $db_in_aa = $ARGV[0];
my $id_in = $ARGV[1];
#
print ("$id_in\n");
#
#
my $dbi = DBI->connect  ("dbi:SQLite:$db_in_aa")
		|| die "Cannot connect: $DBI::errstr";
#
sql_manipulate::dbi_delete_proc ($id_in,$dbi);
$dbi->disconnect;
#
print	(encode ('utf-8',"*** 終了 ***\n"));
# ---------------------------------------------------------

File: sqlite3/node/delete/sqlite3_delete.js
#! /usr/bin/nodejs
//
//	sqlite3_delete.js
//
//					Dec/26/2016
//
// ------------------------------------------------------------
var sqlite3 = require("sqlite3")
var text_manipulate= require ('/var/www/data_base/common/node_common/text_manipulate')

// ------------------------------------------------------------
function delete_command_gen (id_in)
{
	const command = "delete from cities where id = '" + id_in + "'"

	console.log (command)

	return	command
}

// ------------------------------------------------------------
console.log ("*** 開始 ***")

const file_sqlite3 = process.argv[2]
const id_in=process.argv[3]

console.log (file_sqlite3)
console.log (id_in)

var db = new sqlite3.Database (file_sqlite3)

const command = delete_command_gen (id_in)

db.run (command)

db.close ()
console.log ("*** 終了 ***")
// ------------------------------------------------------------
